### PR TITLE
Add views 

### PR DESCRIPTION
**Pull request by dlwub**
In this activity:
- Views have been added for index and show actions of the controllers

### CODE REVIEW

File: app/views/users/index.html.erb [hunk 1]
```diff
@@ -1 +1,7 @@
-<h1>List of users</h1>
\ No newline at end of file
+<div class="container">
+    <ul> 
+        <% @users.each do |user|%>    
+            <%= render "shared/user_data", user: user %>
+        <% end %>
+    </ul>
```
Comment (VICTOR-CHUKWU) [inline app/views/users/index.html.erb:6]: - [ ] kindly make sure your user index page matches the given wireframe
your design :point_down: 
Screenshot from 2023-04-06 17-29-46
wireframe :point_down: 
Screenshot from 2023-04-06 17-30-29

File: app/views/users/show.html.erb [hunk 2]
```diff
@@ -1 +1,17 @@
...
+  <ul>
+    <% @user.recent_posts.each do |post| %>
+      <div class="post">
+        <%= render "shared/post", post: post %>         
+      </div>
+    <% end %>
+  </ul>
+  <%= button_tag link_to "See all posts", user_posts_path(@user) %>   
+</div>
+
```
Comment (VICTOR-CHUKWU) [inline app/views/users/show.html.erb:16]: - [ ] also kindy make your user show page to match the given wireframe
your design :point_down: 
Screenshot from 2023-04-06 17-32-53
wireframe :point_down: 
Screenshot from 2023-04-06 17-33-25

File: app/views/posts/index.html.erb [hunk 3]
```diff
@@ -1 +1,14 @@
...
+    <% @posts.each do |post| %>
+      <div class="post">
+        <%= render "shared/post", post: post %> 
+        <% post.comments.each do |comment| %>
+          <%= render "shared/comments", comment: comment %>
+        <% end %>
+      </div>
+    <% end %>
+  </ul>
+  <button class="post-btn">Pagination</button>
```
Comment (VICTOR-CHUKWU) [inline app/views/posts/index.html.erb:13]: - [ ] kindly make your post index to match the wireframe too
your design :point_down: 
Screenshot from 2023-04-06 17-36-16

wireframe :point_down: 
Screenshot from 2023-04-06 17-35-52

File: app/views/posts/show.html.erb [hunk 4]
```diff
@@ -1,2 +1,11 @@
...
+<div class="individual-post-controller">  
+  <div class="individual-post">
+    <div>Post # <%= @post.id %> by <%= @user.name %></div>
+    <div><%= @post.text %></div>
+    <%= render "shared/comments_likes", post: @post %>    
+    <ul>
+      <% @post.comments.each do |comment| %>
+        <%= render "shared/comments", comment: comment %>
+      <% end %>      
+    </ul>
```
Comment (VICTOR-CHUKWU) [inline app/views/posts/show.html.erb:10]: - [ ] your post show page should match the wire frame kindly fix this
your design :point_down: 
Screenshot from 2023-04-06 17-39-18
wireframe :point_down: 
Screenshot from 2023-04-06 17-38-50

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @dlwub  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by sumairq — APPROVED** [review]
### Hi @dlwub , 



Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

## Highlights
✔️ Good commit history
✔️Linters are passing
✔️Very professional README
✔️Correct Git flow
✔️Descriptive PR
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**